marketplacetests/payment/app.py

```diff
@@ -47,7 +47,6 @@ def create_pin(self, pin):
         Wait(marionette=self.marionette).until(lambda m: 'Create' in self.pin_heading)
         self.marionette.find_element(*self._pin_container_locator).send_keys(pin)
         self.tap_pin_continue()
-        self.wait_for_element_displayed(*self._pin_container_locator)
```

Comment:
I removed this because _pin_container_locator is already displayed and it's something about this that's making this test fail locally. If the click to restore the keyboard follows instantly test passes without problems.